src/main/features/dashboard/views/contact-them.njk

```diff
@@ -0,0 +1,101 @@
+{% extends "layout.njk" %}
+{% from "./macro/caseHistory.njk" import caseHistoryForClaimantClaimDetails %}
+{% from "../../claim/views/macro/about-claim-widget.njk" import aboutClaimWidget %}
+
+{% from "table.njk" import tableStart,tableEnd, row, rowWithMultipleValue %}
+
+{% set headingVisible = false %}
+
+{% if user.id === claim.claimantId %}
+  {% set otherParty = 'defendant' %}
+  {% set otherPartyName = claim.claimData.defendant.name %}
+  {% set phone = claim.response.defendant.mobilePhone %}
+  {% set email = claim.claimData.defendant.email %}
+  {% set contactPerson = claim.response.defendant.name %}
+  {% set address = claim.response.defendant.address if claim.response.defendant.address else claim.claimData.defendant.address %}
+  {% set correspondenceAddress = claim.response.defendant.correspondenceAddress if claim.response.defendant.correspondenceAddress else claim.claimData.defendant.correspondenceAddress %}
+  {% set contactPerson = claim.response.defendant.contactPerson %}
+
+{% elif user.id === claim.defendantId %}
+  {% set otherParty = 'claimant' %}
+  {% set otherPartyName = claim.claimData.claimant.name %}
+  {% set phone = claim.claimData.claimant.mobilePhone %}
+  {% set email = claim.claimantEmail %}
+  {% set address = claim.claimData.claimant.address %}
+  {% set correspondenceAddress = claim.claimData.claimant.correspondenceAddress %}
+  {% set contactPerson = claim.claimData.claimant.contactPerson %}
+
+{% endif %}
+
+{% set heading = 'Contact ' + t(otherParty) %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">
+        <span class="heading-secondary">{{ t(otherParty) | capitalize }}</span>
+        {{ otherPartyName }}
+      </h1>
+      <div class="notice">
+        {% if address %}
+          <h2 class="heading-medium">{{ t('Address') }}</h2>
```

Comment:
This isn't in the prototype:
https://hmcts-cmc-prototype.herokuapp.com/prototypes/features-to-be-built//dashboard/party-details/defendant

Reply:
Had a chat with Ronan and he is ok for address to have address heading and correspondence address for correspondence address. Prototype doesn't have it.